style.css

```diff
@@ -0,0 +1,339 @@
+@import url("https://fonts.googleapis.com/css2?family=Playfair+Display&family=Poppins:wght@300;400;500;600;700;800&display=swap");
+
```

Comment:
I suggest you use mobile to desktop approach by using @media and screen (min-width:375){
then the css for the mobile follows
}